templates/import/import.sample.csv

```diff
@@ -0,0 +1,5 @@
+name   ,question                                                          ,shuffle,min_choices,max_choices,language,choice_1     ,choice_1_score,choice_2         ,choice_2_score,choice_3     ,choice_3_score,choice_4   ,choice_4_score,choice_5,choice_5_score,correct_answer     ,metadata_subject_code,metadata_identifier,metadata_grade
+QCM1000,What is the largest ocean?                                        ,0      ,0          ,1          ,en-US   ,Atlantic     ,              ,Pacific          ,              ,Arctic       ,              ,           ,              ,        ,              ,choice_2           ,GEO_b                ,12                 ,5
+QCM2000,What is the <strong>largest</strong> mountain?                    ,1      ,0          ,1          ,en-US   ,Mount Olympus,0             ,Mount Kilimanjaro,2             ,Mount Everest,5             ,Pichu Pichu,3             ,        ,              ,choice_3           ,GEO_b                ,15                 ,
+QCM3000,Which is the <strong>smallest</strong> planet of our solar system?,       ,           ,           ,        ,Earth        ,4             ,Uranus           ,2             ,Jupiter      ,1             ,Neptune    ,3             ,Mercury ,5             ,"choice_5"         ,                     ,35                 ,
+QCM4000,Which of those are cities of Italy?                               ,       ,           ,           ,        ,Venice       ,1             ,Rome             ,1             ,New York     ,0             ,Paris      ,0             ,        ,0             ,"choice_1,choice_2","""geography"""      ,20                 ,"1,2"
```

Comment:
If you remove empty spaced after quotes - github will prettify your csv 😄 
![image](https://user-images.githubusercontent.com/59471572/109239086-bbfbf580-77e5-11eb-946e-68bd4615915a.png)

```suggestion
QCM3000,Which is the <strong>smallest</strong> planet of our solar system?,       ,           ,           ,        ,Earth        ,4             ,Uranus           ,2             ,Jupiter      ,1             ,Neptune    ,3             ,Mercury ,5             ,"choice_5",                     ,35                 ,
QCM4000,Which of those are cities of Italy?                               ,       ,           ,           ,        ,Venice       ,1             ,Rome             ,1             ,New York     ,0             ,Paris      ,0             ,        ,0             ,"choice_1,choice_2","""geography""",20                 ,"1,2"
```

Reply:
Indeed, but it is a dilemma, cause on the other hand it become hard to read on the text editor :/ 

So I preferred to make it easier to read in the text editor. Is it ok for you?